README.md

```diff
@@ -110,8 +105,6 @@ Next we are going to add a form onto the Contact page, allowing people to email
 Note: You only need to type the **git push origin master:gh-pages** command once when you initially publish your website. If you make updates to your website at home and want to publish the changes, you can stop after **git push origin master**.
 
 ## Homework
-Did you love working with code? If you want to learn more on your own, we have suggestions for you. 
+Did you love working with code? If you want to learn more on your own, we have suggestions for you.
 1.  Create an account on http://www.codecademy.com/
 2.  Complete this course on Codecademy: http://www.codecademy.com/courses/web-beginner-en-zmn0b/0/1?content_from=make-an-interactive-website%3Ayour-first-program
-
```

Comment:
What about adding instructions to fill out the survey here too?

Reply:
I think that's a good idea. Right now, though, our survey has language specific to Jewelbots: https://docs.google.com/forms/d/e/1FAIpQLSez_eKdTOfZsfv_IrxAFOaW20Hp5o7q05uosXrGMw_2OTepIQ/viewform 

I think we could make it generic so it works for both sessions, and after that I can add a link to it at the end of the Web Dev worksheet. Who does the editing of the survey? @cschneide143 ?